nrf_security/doc/api.rst

```diff
@@ -3,6 +3,10 @@
 API documentation
```

Comment:
There are more files in this folder that need the ToC.

Reply:
fixed